webapp/packages/plugin-connections/src/ConnectionForm/Options/Options.tsx

```diff
@@ -220,17 +222,17 @@ export const Options: TabContainerPanelComponent<IConnectionFormProps> = observe
                   {translate('connections_connection_driver')}
                 </Combobox>
                 {configurationTypes.length > 1 && (
-                  <FormFieldDescription label={translate('connections_connection_configuration')} tiny>
+                  <FormFieldDescription label={configurationTypeLabel} tiny>
                     <Container gap>
-                      <RadioGroup name="configurationType" state={optionsPart.state}>
+                      <RadioGroup labelledBy={configurationTypeLabel} name="configurationType" state={optionsPart.state}>
```

Comment:
labelledBy is for [aria-labelled-by](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby) which is waiting for an id of an element. So, in order to use it properly, you need to provide the ID of a <label> with description. Since that <label> is hidden inside a FormFieldDescription, we can use just aria-label here, it will not be rendered and will provide an information about this control to assistive tech